### DISCUSSION TITLE
Collapsible sections

### DISCUSSION THREAD

**Discussion by emmaehrlich** [willfaught/paige#73] [General]
Is there a way to create collapsible sections within a post?

For example, I can make what I want by adding this to index.html within the "public" folder
<details>
    <summary>
      Collapsible section
      <span class="icon">👇</span>
    </summary>
    <p>
      With content inside
    </p>
  </details>

but it gets deleted every time I restart the server.

Maybe there's a way to modify style-first.css to make it so that part of the html file will be safe from being changed by the CSS (or whatever is overwriting my html)?

**Comment by willfaught** [comment]
You need to enable unsafe Markdown rendering in yoursite/hugo.toml:

```toml
[markup.goldmark.renderer]    
unsafe = true
```

**Comment by emmaehrlich** [comment]
That works - thank you!
